### DISCUSSION TITLE
Netbox 3.4.x needs rq 1.13.0, newer rq deprecates use_connection()

### DISCUSSION THREAD

**Discussion by mtinberg** [netbox-community/netbox#13284] [General]
I just updated Netbox to 3.4.10 from 3.4.7 on an RHEL7 host using the rh-python38 and ran into a couple of python libraries that I needed to pin to older versions with pip constraints

1. urllib3==1.26.16 as urllib3 v2.+ requires OpenSSL 1.1.1+ and EL7 only has OpenSSL 1.0.1k (patched)
2. rq==1.13.0 as 1.14+ includes [Stop calling deprecated use_connection()](https://github.com/rq/rq/commit/d5bde117c2f5477b5d0ba9846f2a7c4480ca1854) which is used by rqworker, This is probably been refactored in 3.5.x, I haven't checked but 3.5 came out after this change to rq.

HTH

**Comment by jcollie** [comment]
Duplicate of #12415
